scripts/gulp/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    "prefer-arrow-callback": "off"
```

Comment:
The lint checks were passing on master (in CI builds at least) in previous commits. Has something changed on this branch or do you see an issue on master as well?

Reply:
It was related to my IDE , I removed the rule.